docs/en/user-guide/editing/connect_wastewater_network_elements.rst

```diff
@@ -18,23 +18,34 @@ How to connect reaches to nodes or reaches to reaches
 * It does not matter, which layer is selected
 * The **vw_tww_reach** layer has to be in edit mode
 * Click the **Connect wastewater network elements** button
-* If you move the mouse over an reach, that can be connected, it is marked: you can click
+* If you move the mouse over a reach, that can be connected, it is marked (orange color): you can click
 * Move the mouse to the network element, to which the first object will be connected (again: selectable objects get marked) and click again
 
 .. figure:: images/connecting_1.jpg
 
+.. note:: the topology-layers are turned on in the screenshot to show the effect of the connection.
+
 * A window appears, where you can choose, which connection will be defined. If you connect a reach, the window looks like this:
 
 .. figure:: images/connecting_2.jpg
 
+.. note:: there is no options 'Reach Point From' in this example, because in a connection from reach to reach, it is not allowed to connect a From-reach point to another reach.
+
+
 * If everything is ok, you will see a message in a blue bar on top of the map-window
 
 .. figure:: images/connecting_3.jpg
 
+
+* To show the new connection, save your edits, click on the TWW-SQL-Button (refresh network topology) and refresh the screen. Now you can see the new connection in the topology-layers
+
+.. figure:: images/connecting_4.jpg
+
+
 Video Tutorial
 --------------
 
-* See `this video tutorial (QGIS version 2): <https://vimeo.com/171536774>`_
+* See `this video tutorial (QGEP and QGIS version 2): <https://vimeo.com/171536774>`_
```

Comment:
This will create a keyword warning

Reply:
@ponceta We have to think anyway where to place videos - there is another one https://vimeo.com/162978741

Reply:
Changed. That it's not possible to mention QGEP is quite strange. It's good to have hint's about old text, but it should be possible to mention QGEP as preversion of TWW. Perhaps it is possible to relax this rule...